src/main/plugins/org.dita.xhtml/xsl/xslhtml/dita2htmlImpl.xsl

```diff
@@ -1078,7 +1079,7 @@
         <xsl:if test="ancestor::*[contains(@class, ' topic/title ')]/parent::*[contains(@class, ' topic/topic ')]">
           <xsl:choose>
             <!-- Not the first one in a title -->
-            <xsl:when test="generate-id(.) != generate-id(key('tm', .)[1])">skip</xsl:when>
+            <xsl:when test="generate-id(.) != generate-id($root/key('tm', .)[1])">skip</xsl:when>
```

Comment:
It probably is, but I'm in the habit of using this form because if the third argument to `key()` is an empty sequence, it is a static error, whereas `$el/key('foo', 'bar')` will return an empty sequence if `$el` is an empty sequence.

In this case, though, `$root` can't be empty, so it's not a problem to change.